### ISSUE TITLE
Add support for calibre receipes 

### ISSUE THREAD

**Issue by skrawani** [gbkel/kindlefy#4]
@guilhermebkel 
I want to inquire if there are any plans to add support for **calibre recipes**. As a regular user of a fork with calibre recipe support, I believe that adding this feature would greatly improve the functionality of the platform.

I have made some changes to the code to support calibre recipes, and I would like to introduce these changes to the master branch if there is interest in implementing this feature. Please let me know if this is something that is open for discussion.

**Comment by guilhermebkel** [comment]
Hello @skrawani! Thanks for getting in touch!

It is awesome to know you were able to implement a feature by yourself using this code base :)

Being honest I didn't know about **calibre recipes** 🤔 But I'm curious to give a look at it. Do you recommend any article?

I'm open to talk about this feature :)

**Comment by skrawani** [comment]
> A recipe is a set of instructions that teach calibre how to convert an online news source, such as a magazine or a blog, into an e-book. A recipe is essentially [Python](https://www.python.org/) code. As such, it is capable of converting arbitrarily complex news sources into e-books. At the simplest level, it is just a set of variables, such as URLs, that give calibre enough information to go out onto the Internet and download the news.

@guilhermebkel Yeah you can check this out https://manual.calibre-ebook.com/news.html
